### PR TITLE
Update France holidays, localization

### PR DESCRIPTION
**Pull request by KJhellico**
## Proposed change

- migrate to holidays groups
- fix some holidays date / starting date
- en_US, fr (default), uk localization

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good (`make pre-commit`)
- [x] I've added tests to verify that the new code works and all tests pass locally (`make test`, `make tox`)
- [x] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/countries/france.py [hunk 1]
```diff
@@ -49,99 +49,129 @@ class France(HolidayBase):
         "Wallis-et-Futuna",
     ]
 
-    def __init__(self, **kwargs):
-        # Default subdivision to Métropole; prov for backwards compatibility
-        if not kwargs.get("subdiv", kwargs.get("prov")):
-            kwargs["subdiv"] = "Métropole"
-        HolidayBase.__init__(self, **kwargs)
+    def __init__(self, *args, **kwargs):
```
Comment (arkid15r) [inline holidays/countries/france.py:52]: I believe we should keep that default subdiv backward compatibility piece of code.
At least let's run it through a deprecation process steps?

File: holidays/locale/en_US/LC_MESSAGES/FR.po [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+#. Feast of Victor Schoelcher.
+#: holidays/countries/france.py:126
+msgid "Fête de Victor Schoelcher"
+msgstr "Feast of Victor Schoelcher"
+
+#. Abolition of slavery.
+#: holidays/countries/france.py:129
+msgid "Abolition de l'esclavage"
+msgstr "Abolition of slavery"
```
Comment (arkid15r) [inline holidays/locale/en_US/LC_MESSAGES/FR.po:97]:
```suggestion
msgstr "Abolition of Slavery"
```

File: holidays/locale/en_US/LC_MESSAGES/FR.po [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+#. Feast of Saint Peter Chanel.
+#: holidays/countries/france.py:168
+msgid "Saint Pierre Chanel"
+msgstr "Feast of Saint Peter Chanel"
+
+#. Festival of the territory
+#: holidays/countries/france.py:170
+msgid "Fête du Territoire"
+msgstr "Festival of the territory"
```
Comment (arkid15r) [inline holidays/locale/en_US/LC_MESSAGES/FR.po:127]:
```suggestion
msgstr "Festival of the Territory"
```

**Review by arkid15r — COMMENTED** [review]
LGTM

**Review by arkid15r — COMMENTED** [review]
LGTM

twice? :)

File: holidays/countries/france.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/france.py:52]: There is no specific holidays for "Métropole", It's like a synonym for "continental France". So if someone has used `holidays.FR(subdiv="Métropole")`, they will receive the same holidays set.

Comment (arkid15r) [reply holidays/countries/france.py:52]: Well, this makes it easier then 👍

**Comment by arkid15r** [comment]
Merged into beta!

**Pull request merged**